tests/services-aerospace.nix

```diff
@@ -43,5 +65,20 @@ in
     grep 'alt-j = "focus down"' $conf
     grep 'alt-k = "focus up"' $conf
     grep 'alt-l = "focus right"' $conf
+
+    grep 'check-further-callbacks = false' $conf
+    grep 'run = "move-node-to-workspace m"' $conf
+    grep 'app-id = "Another.Cool.App"' $conf
+    grep 'during-aerospace-startup = false' $conf
```

Comment:
It would be great if you also check that unspecified options don't appear in the config e.g. the `window-title-regex-substring`.

Reply:
I don't think default null values have a way of somehow entering TOML files, since they have the concept of null = omitted. Hence I'm not sure how to feasibly test that in this specific implementation. maybe we check that `check-further-callbacks = false` is not present before the line `run = "layout floating"`

Reply:
I think it's fine to do a negative assertion for fields that don't show up in the config file, you can also be explicit about the value by setting them to `null` intentionally rather than relying on the default value of the option

You can see how I did them here: https://github.com/LnL7/nix-darwin/commit/fd6660cb9182fde5e593246e311dc3c2dd4b9d13

Reply:
thanks for the hint!
I'll try to tackle that when I get some time.